nupic/algorithms/anomaly.py

```diff
@@ -84,22 +93,15 @@ def __init__(self, slidingWindowSize=None, mode=MODE_PURE, binaryAnomalyThreshol
               models probability of receiving this value and anomalyScore
           - "weighted" - "pure" anomaly weighted by "likelihood"
               (anomaly * likelihood)
+          - OR type function and will be used directly to compute score
```

Comment:
Also, I am not crazy about overloading args, such as mode, to pass a function instead of a string. I think that if I was working on this, I would likely add a new optional kwarg: e.g., `computeAnomaly=None`. A user that wishes to pass his own computeAnomaly callable, would invoke the constructor with `mode=None` and `computeAnomaly=<user's callable>`